src/D2L.CodeStyle.Analyzers/DependencyScope/DependencyScopeAnalyzer.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using D2L.CodeStyle.Analyzers.Extensions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Text;
+
+namespace D2L.CodeStyle.Analyzers.DependencyScope {
+
+	[DiagnosticAnalyzer( LanguageNames.CSharp )]
+	public sealed class DependencyScopeAnalyzer : DiagnosticAnalyzer {

```

Comment:
If this is the only purpose of the analyzer long-term (which would be fine) I'd go for a more specific name, like `OnlyWebRequestAnalyzer`